data/coalition/coalition missions.txt

```diff
@@ -143,3 +143,315 @@ mission "Coalition License Patch"
+			label mistress
+			`	He asks that you go inside the vehicle, saying, "Assist Lugglop in his research, my mistress will. Meet her in the mansion, you must."`
+			`	The short trip is uneventful, with you managing to have a seat without much discomfort in the Arachi vessel, though you still don't know what the extra wheels are for.`
+			`	The entrance to the mansion isn't as castle-like as the walls, though it all still looks strangely pompous. Up close it almost looks like all the several parts and pods of an Arach Hulk or two were crumpled together into one single building.`
+			`	He guides you through the mansion, as you pass through several other servants, some rooms where he mentions that members of House Mallob are having meetings, and even meet a group of Saryd and Kimek students on the way, here to study with the shipwrights of House Mallob.`
+			`	Finally, he brings you to a room on the third floor, claiming that your passenger is waiting inside. You enter, only to see a Kimek glancing out the window.`
+			choice
+				`	"So, is my passenger crawling around somewhere? In some corner hidden in a web?"`
+				`	"A Kimek? I thought this was an Arach House."`
+			`	The butler ignores what you said, and introduces you to the Kimek in question. "Marry into House Mallob at a young age, miss Pichiie did. A former student on the university here, she was."`
+			`	Finally, the reason to why you had to come all the way here to this part of <origin> becomes clear, as Pichiie mentions she would like to hear your stories about where you first got the Yottrite in the first place, before boarding your ship to head to <planet>. You do your best to keep it as brief as possible, but she constantly keeps asking more and more questions, ranging from how you acquired a Jump Drive to trying to recall places you've been to by memory.`
+			`	About an hour after the sun sets, she's finally satisfied, and after saying her goodbyes to other members of the House, you're both brought to your ship.`
+				accept
+	on complete
+		event "yottrite research break" 165
+		payment 200000
+		conversation
+			`As with the Saryd twins, Lugglop greets Pichiie and tells her of a hotel he prepared for her time here.`
+			`	"Ready, my team is now, Captain," he says, after Pichiie leaves with her luggage. "Enrich and research samples as needed, for the next few months we will. Contact you again, I will, should the need for your services arise once again."`
+			`	He hands you <payment>, and walks away.`
+
+
+
+event "yottrite research break"
+
+
+
+mission "Coalition Yottrite 8"
+	name "One Last Sample"
+	description "Acquire one sample of Yottrite, and deliver it to <destination>, so that Lugglop has some for his presentation to the Coalition's scientific community."
```

Comment:
```suggestion
	description "Acquire one sample of Yottrite and deliver it to <destination> so that Lugglop has some for his presentation to the Coalition's scientific community."
```
Double predicate and a dependent clause following independent clause; no comma for both.